src/test/java/com/shapesecurity/shift/es2016/parser/miscellaneous/WhitespaceTest.java

```diff
@@ -98,7 +98,7 @@ public void testWhitespace() throws JsError {
 
         testScript("throw /* \u202a */ e", new ThrowStatement(new IdentifierExpression("e")));
 
-        testScript("new\u0020\u0009\u000B\u000C\u00A0\u1680\u180E\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u202F\u205F\u3000\uFEFFa",
+        testScript("new\u0020\u0009\u000B\u000C\u00A0\u1680\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u202F\u205F\u3000\uFEFFa",
```

Comment:
This is a separate issue, updating Unicode data to recognise fewer whitespace characters. Please split it out into its own PR.